dashboard/config/levels/custom/music/hoai2025-music-adlib-2.level

```diff
@@ -1,18 +1,45 @@
 <Music>
   <config><![CDATA[{
+  "published": true,
   "game_id": 70,
   "created_at": "2025-09-10T22:40:27.000Z",
   "level_num": "custom",
   "user_id": 1,
   "properties": {
-    "preload_asset_list": null,
     "level_data": {
       "library": "launch2024",
       "aiCodeGenerate": "true",
-      "aiCodeGenerateAdlib": "complex"
-    }
+      "aiCodeGenerateAdlib": {
+        "id": "sounds",
+        "template": "Please make a song with {sounds} sounds in order.",
```

Comment:
Do we want `id` to be intentionally decoupled from `template`? Or should the ID always be comprised of the adlib fields joined by dashes (in which case, should we remove the `id` field altogether)?